### PR TITLE
LSP4J 0.20

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

**Review by rubenporras — APPROVED** [review]

**Comment by mickaelistria** [comment]
LSP4E CI infra is stuck: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/2709 . Let's wait for CI to validate this PR before we merge.

**Pull request merged**
